modules/fun.js

```diff
@@ -0,0 +1,450 @@
+    msg.channel.send((Math.random() < 0.8 ? `pass` : `fail`));
+  }
+  const exclude = ['121033996439257092', '164784857296273408'];// IDK where these are so hardcoded they shall currently remain.
+  const roll = Math.random();
+  if (roll < 0.3 && !msg.author.bot && !exclude.includes(msg.channel.id)) {
+    // let banana = /[bß8ƥɓϐβбБВЬЪвᴮᴯḃḅḇÞ][a@∆æàáâãäåāăȁȃȧɑαдӑӓᴀᴬᵃᵅᶏᶐḁạảấầẩẫậắằẳẵặ4Λ]+([nⁿńňŋƞǹñϰпНhийӣӥѝνṅṇṉṋ]+[a@∆æàáâãäåāăȁȃȧɑαдӑӓᴀᴬᵃᵅᶏᶐḁạảấầẩẫậắằẳẵặ4Λ]+){2}/ig;
+    if (msg.content.toLowerCase().includes(`bananas`)) {
+      if (roll < 0.1) {
+        msg.channel.send({ files: [new Discord.AttachmentBuilder('media/buttermelonsMan.jpeg')] }).catch(u.errorHandler);
+      } else {
+        msg.channel.send(`*buttermelons`).catch(u.errorHandler);
+      }
+    } else if (msg.content.toLowerCase().includes(`banana`)) {
+      if (roll < 0.06) {
+        msg.channel.send({ files: [new Discord.AttachmentBuilder('media/buttermelonPile.png')] }).catch(u.errorHandler);
+      } else if (roll < 0.1) {
+        msg.channel.send({ files: [new Discord.AttachmentBuilder('media/buttermelonMan.jpeg')] }).catch(u.errorHandler);
+      } else {
+        msg.channel.send(`*buttermelon`).catch(u.errorHandler);
+      }
+    }
+  }
+}
+/**
+ * function slashFunQuote
+ * @param {Discord.ChatInputCommandInteraction} int a /fun quote interaction
+ */
+async function slashFunQuote(int) {
+  const url = `https://api.forismatic.com/api/1.0/?method=getQuote&format=json&lang=en`;
+  const response = await axios({ url, method: `get` }).catch((/** @type {axios.AxiosError} */ e) => {
+    throw new Error(`quote command error:${e.status}`);
+  });
+  const data = response.data;
+  if (data) {
+    const randomQuote = data;
+    return int.editReply(`> ${randomQuote.quoteText}\n> - ${randomQuote.quoteAuthor}`);
+  } else {
+    return int.editReply(`> A developer uses dark mode because bugs are attracted to light, \n` +
+    `> but wouldn't that put the bugs in the code instead of the background?\n` +
+    `> - ChainSword20000`);
+  }
+}
+/**
+ * function namegame
+ * @param {Discord.ChatInputCommandInteraction} int a /fun namegame interaction
+ */
+async function slashFunNamegame(int) {
+  let nameArg = int.options.getString(`name`);
+  try {
+    if (!nameArg) nameArg = int.user.displayName;
+    nameArg = nameArg.replace(/[^a-zA-Z]/g, '_');// just ABCabc etc, numbers were causing problems.
+    nameArg = nameArg.split(`_`)[0];// and just one segment
+    const name = nameArg;
+    const url = `https://thenamegame-generator.com/lyrics/${name}.html`;
+    const response = await axios({ url, method: `get` }).catch((/** @type {axios.AxiosError} */ e) => {
+      int.editReply(`Could not generate lyrics for ${name}.\nPerhaps you can get it yourself from https://thenamegame-generator.com.`);
+      throw new Error(`namegame command error:${e.status}`);
+    });
+    const data = response.data;
+    if (data) {
+      const profanityFilter = require(`profanity-matcher`);
```

Comment:
to the top

Reply:
I thought I moved all of these. perhaps the commit got lost somewhere... found it, I put it in the emoji branch... I got /emoji working and tested though now, so I suppose that can be merged back in...